api/src/main/java/com/pip/chatbot/ApplicationConfiguration.java

```diff
@@ -46,40 +46,34 @@ public CountriesRepository countriesRepository(DSLContext dsl) {
     }
 
     @Bean
-    public JokesRepository noAuthJokeRepository(DSLContext dsl) {
+    public JokesRepository jokesRepository(DSLContext dsl) {
         return new JokesRepository(dsl);
     }
 
     @Bean
-    @Qualifier("Joke")
-    public ModelMapper modelMapperJoke() {
-        PropertyMap<JokeApi, Joke> propertyMap = new PropertyMap<JokeApi, Joke>() {
+    public ModelMapper modelMapper() {
+        ModelMapper mapper = new ModelMapper();
+        mapper.getConfiguration().setMatchingStrategy(MatchingStrategies.STRICT);
+
+        mapper.addMappings(new PropertyMap<JokeApi, Joke>() {
             @Override
-            public void configure() {
+            protected void configure() {
                 map().setJoke(source.getJoke());
                 map().setCategory(source.getCategory());
             }
-        };
-        ModelMapper modelMapper = new ModelMapper();
-        modelMapper.addMappings(propertyMap);
-        return modelMapper;
-    }
-
-    @Bean
-    @Qualifier("Mark")
-    public ModelMapper modelMapperMark() {
-        PropertyMap<MarkApi, Mark> propertyMap = new PropertyMap<MarkApi, Mark>() {
+        });
+        mapper.addMappings(new PropertyMap<MarkApi, Mark>() {
             @Override
-            public void configure() {
+            protected void configure() {
```

Comment:
Trzeba robić mapowanie dla tak samo nazywających się pól? Na pewno da się go tak skonfigurować żeby to ogarnął:)

Reply:
chyba ok